x-pack/plugins/uptime/public/components/fleet_package/custom_fields.tsx

```diff
@@ -122,6 +124,34 @@ export const CustomFields = memo<Props>(
                   />
                 </EuiFormRow>
               )}
+              <EuiSpacer size="s" />
+              {isBrowser && (
+                <EuiCallOut
+                  title={
+                    <FormattedMessage
+                      id="xpack.uptime.createPackagePolicy.stepConfigure.monitorIntegrationSettingsSection.monitorType.browser.warning.description"
+                      defaultMessage='Browser monitoring requires using the "complete" variant of the elastic–agent docker container. Other distributions of elastic that agent will not work correctly with it because they lack the required browser and other binaries. For more information see our {link}'
```

Comment:
```suggestion
                      defaultMessage='Browser monitoring requires using the "complete" variant of the elastic–agent docker container. Other distributions of elastic that agent will not work correctly with it because they lack the required browser and other binaries. For more information see our {link}.'
```

Also, this sentence:

```
Other distributions of elastic that agent will not work correctly with it because they lack the required browser and other binaries. 
```

This reads weirdly to me. Are we saying that "distributions of elastic" might not work correctly?

Reply:
This copy was a placeholder. Drew just provided the final version.

Reply:
Whoops, sorry, I saw that he commented and was confused why they didn't match. Should've put 2 + 2 together!